### DISCUSSION TITLE
Differentiating Additional Fuel Dispensing POIs

### DISCUSSION THREAD

**Discussion by julcnx** [osmandapp/OsmAnd#18142] [Ideas]
In countries like Thailand and other developing nations, where motorcycles are prevalent, there are various types of fuel distribution points beyond the typical retail fuel stations (`amenity=fuel`). These alternative categories of fuel dispensing may warrant distinct tag combinations:

- Drummed fuel shops (often tagged with shop=fuel).
- Fuel vending machines (often tagged with amenity=vending_machine + vending=fuel).
- General convenience stores selling small bottles (1L/1.5L, often tagged with shop=convenience + vending=fuel).

These additional categories often come with higher prices and limited availability. Therefore, it would be highly beneficial to visually differentiate them, perhaps using unique icons or colors.

We are currently engaging in a discussion about this topic with the global community at the following link: https://community.openstreetmap.org/t/differing-approaches-to-tagging-vending-machines-for-motor-fuel-dispensing-have-caused-confusion/103421

We'd love to hear your perspective on this matter!

**Comment by DmitryAlexei** [comment]
Related https://github.com/osmandapp/OsmAnd/issues/17967
